tests-e2e/num_live_test.go

```diff
@@ -122,7 +122,6 @@ func TestNumLive(t *testing.T) {
 	m.MatchResponse(t, res, m.MatchRoomSubscriptionsStrict(map[string][]m.RoomMatcher{
 		roomID: {
 			m.MatchNumLive(2),
-			// TODO: should we be including event ID 2 given timeline limit is 2?
```

Comment:
Don't really follow this---why have we now deeemed this behaviour desirable?

Reply:
I've removed this mostly because A) it's always been just for the initial limit and B) I originally had the test in this file before realising it needed to be an integration test.